06/tasks/task_26_is_palindrome_sentence.py

```diff
@@ -4,4 +4,7 @@ def is_palindrome_sentence(s: str) -> bool:
     """
     Vrátí True, pokud je věta palindrom (ignoruje mezery, interpunkci a velikost písmen).
     """
+# creates a list consisting only of lowercase letters of the original string
+    s_list = [char.lower() for char in list(s) if char.isalpha()]
```

Comment:
```suggestion
    s_list = [char.lower() for char in s if char.isalpha()]
```
neni nutne volat `list()`.

Reply:
JJ, máš pravdu, opraveno.